hack/testing/check-EFK-running.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+
```

Comment:
I am not a huge fan of -x, I find it makes the output a bit too verbose by default. But I do find it can be useful to enable it with a verbose flag.

You may also want to set -u (variables have to be declared) and '-o pipefail' as extra precautions